### ISSUE TITLE
where can I get the trained models like MARS on HMDB51-2,3

### ISSUE THREAD

**Issue by runtoforever** [craston/MARS#37]
hello, I'm trying to implement on HMDB51-2,3 datasets, using HMDB51-1 model weight for pre-training.The accuracy obtained was less than 76% on test set. I want to know whether you still have a trained model.

**Comment by craston** [comment]
The trained models for MARS HMDB51-1,2,3 are [here](https://drive.google.com/drive/folders/1OVhBnZ_FmqMSj6gw9yyrxJJR8yRINb_G?usp=sharing)
